doc/cdrwtool.1

```diff
@@ -135,11 +135,29 @@ variable and fixed packet sizes respectively.
 .IP "\fB\-o \fIoffset\fP"
 Set write offset.
 
+.SH BUGS
+Many modern drives refuse on the preparations to format new,
+blanked, or appendable CD-RW media. This causes a message like
+.nf
+  "Command failed: 55 ... - sense ..."
+.fi
+.br
+The remedy is to use a CD-capable burn program for writing a session
+and closing the medium. For example by using any of "cdrecord", 
```

Comment:
I would suggest to use better manpage/troff formatting (instead of quotes):
```suggestion
and closing the medium. For example by using any of \fBcdrecord\fP(1), 
```